app.rb

```diff
@@ -0,0 +1,124 @@
+require './student'
+require './teacher'
+require './rental'
+require './classroom'
+require './person'
+require './book'
+
+class App
+  def initialize
+    @people = []
+    @books = []
+    @rentals = []
+    @classroom = Classroom.new('Grade 10')
+  end
+
+  def list_all_books
+    if @books.empty?
+      puts 'Oops! the library is empty, please add books'
+      return
+    end
+    @books.each do |book|
+      puts "Title: #{book.title}, Author: #{book.author}"
+    end
+  end
+
+  def list_all_people
+    if @people.empty?
+      puts 'Oops! no people registered, please add people'
+    else
+      @people.each do |person|
+        puts "[#{person.class}] Name: #{person.name}, ID: #{person.id}, Age: #{person.age}"
+      end
+    end
```

Comment:
- [ ] When listing people, the age is supposed to show a number and for the teacher, instead of showing a number for age, it instead shows the specialization. Kindly take a look at the screenshot below or better you can test the app and see what shows up. 

<img width="574" alt="Screenshot 2023-03-25 at 1 03 25 PM" src="https://user-images.githubusercontent.com/50941074/227719184-b7d816f4-a3d1-4e0e-b16a-7ad14f83b558.png">

- [ ] Kindly rearrange the right data to show in the right spot 👍🏽

Reply:
- [ ] Well resolved 👍🏽 
<img width="365" alt="Screenshot 2023-03-25 at 15 27 00" src="https://user-images.githubusercontent.com/58692590/227723359-67bca3a0-5b50-41ce-9ac6-0469fb01c199.png">